functions/_fzf_search_processes.fish

```diff
@@ -0,0 +1,21 @@
+function _fzf_search_processes --description "Search pid for all running commands"
+    set fzf_arguments --multi --ansi $fzf_processes_opts
+    set token (commandline --current-token)
+    # expand any variables or leading tilde (~) in the token
+    #set expanded_token (eval echo -- $token)
+
+    set pid_selected (
+        ps -A -opid,command | \
```

Comment:
Check this out from the ps manpage
```
-c      Change the “command” column output to just contain the executable name, rather than the full command line.
```
should we use it?

Reply:
I intentionally didn't use it. I think being able to list and filter based on the command line arguments might be usable. 
Putting just the command will leave us with a lot of pretty opaque processes for things like shell scripts, ssh sessions etc....

Reply:
good point! I agree. Your ssh sessions argument convinced me.